src/ModalTitle.js

```diff
@@ -1,31 +1,28 @@
 import classNames from 'classnames';
 import React from 'react';
+import PropTypes from 'prop-types';
 import elementType from 'prop-types-extra/lib/elementType';
+import divWithHeadingClass from './utils/divWithHeadingClass';
 
-import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
+import { createBootstrapComponent } from './ThemeProvider';
 
-const propTypes = {
-  as: elementType
-};
-
-const defaultProps = {
-  as: 'h4'
-};
+const DivStyledAsH4 = divWithHeadingClass('h4');
```

Comment:
so the idea is that this looks more or less per the examples, but doesn't insert a weird semantic header component? cool!